src/games/PlayerVsPlayerGame.js

```diff
@@ -1,9 +1,26 @@
+var $ = require('jquery')
+
 function PlayerVsPlayerGame(ui) {
   this.ui = ui;
 }
 
-PlayerVsPlayerGame.prototype.play = function() {
-  this.ui.displayTurn("X");
+PlayerVsPlayerGame.prototype.play = function(callback) {
+  var playerMarker = this.ui.displayTurn("X");
+  $(".spot").on("click", (e) => {
+    var id = callback(e);
```

Comment:
It seems like you might not need to pass a callback in here. I think the spotClicked function in `TicTacToe` should probably live in the `UI` (since it directly interfaces with the HTML), and you already have access to the `UI` here anyways.